saleor/dashboard/page/decorators.py

```diff
@@ -0,0 +1,16 @@
+from functools import wraps
+
+from django.shortcuts import get_object_or_404
+
+from ...page.models import Page
+from .modals import modal_protected_page
+
+
+def unprotected_page_required(view):
```

Comment:
Do we really need the decorator for that purpose? In case of `page_delete` view we could check `page.is_protected` in the view. We could consider adding such decorator when we have more views requiring it.